File: scripts/Scratch.java
import java.io.File;
public class Scratch {
    public static void main(String[] args) {
        File parent = null;
        try {
            File f = new File(parent, "foo.bin");
            System.out.println(f.getAbsolutePath());
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

File: scratch.kt
fun main() {
    val parent: java.io.File? = null
    try {
        val f = java.io.File(parent, "gemma-2b-it-cpu-int4.bin")
        println(f.absolutePath)
    } catch(e: Exception) {
        println("Crash: " + e.message)
    }
}
